### PR TITLE
add docs

### PR DESCRIPTION
**Pull request by pacman100**
### What does this PR do?
1. Adding HF docs

### CODE REVIEW

File: docs/quicktour.mdx [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+# 'complaint'
+```
+
+## Launching your distributed script
+
+PEFT models work with 🤗 Accelerate out of the box. 
+Use 🤗 Accelerate for Distributed training on various hardware such as GPUs, Apple Silicon devices etc during training.
+Use 🤗 Accelerate for inferencing on consumer hardware with small resources.
+
+### Example of PEFT model training using 🤗 Accelerate's DeepSpeed integration
```
Comment (stevhliu) [inline docs/quicktour.mdx:107]:
```suggestion
### Train with 🤗 Accelerate's DeepSpeed integration
```

File: docs/quicktour.mdx [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+  CPU Memory before entering the eval : 19411
+  CPU Memory consumed at the end of the eval (end-begin): 0
+  CPU Peak Memory consumed during the eval (max-begin): 0
+  CPU Total Peak Memory consumed during the eval (max): 19411
+  accuracy=100.0
+  eval_preds[:10]=['no complaint', 'no complaint', 'complaint', 'complaint', 'no complaint', 'no complaint', 'no complaint', 'complaint', 'complaint', 'no complaint']
+  dataset['train'][label_column][:10]=['no complaint', 'no complaint', 'complaint', 'complaint', 'no complaint', 'no complaint', 'no complaint', 'complaint', 'complaint', 'no complaint']
+  ```
+
+### Example of PEFT model inference using 🤗 Accelerate's Big Model Inferencing capabilities
```
Comment (stevhliu) [inline docs/quicktour.mdx:166]:
```suggestion
### Inference with 🤗 Accelerate's Big Model Inference
```

File: docs/quicktour.mdx [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+
+PEFT models work with 🤗 Accelerate out of the box. 
+Use 🤗 Accelerate for Distributed training on various hardware such as GPUs, Apple Silicon devices etc during training.
+Use 🤗 Accelerate for inferencing on consumer hardware with small resources.
+
+### Example of PEFT model training using 🤗 Accelerate's DeepSpeed integration
+
+DeepSpeed version required `v0.8.0`. An example is provided in `~examples/conditional_generation/peft_lora_seq2seq_accelerate_ds_zero3_offload.py`. 
+  a. First, run `accelerate config --config_file ds_zero3_cpu.yaml` and answer the questionnaire. 
+  Below are the contents of the config file.
```
Comment (stevhliu) [inline docs/quicktour.mdx:111]:
```suggestion
You'll need DeepSpeed version `v0.8.0` for this example. Feel free to check out the full example [script](https://github.com/huggingface/peft/blob/main/examples/conditional_generation/peft_lora_seq2seq_accelerate_ds_zero3_offload.py) for more details!

1. Run `accelerate config --config_file ds_zero3_cpu.yaml` and answer the questionnaire to setup your environment. 
Below are the contents of the config file.
```

File: docs/quicktour.mdx [hunk 4]
```diff
@@ -0,0 +1,300 @@
...
+  main_training_function: main
+  megatron_lm_config: {}
+  mixed_precision: 'no'
+  num_machines: 1
+  num_processes: 1
+  rdzv_backend: static
+  same_network: true
+  use_cpu: false
+  ```
+  b. run the below command to launch the example script
```
Comment (stevhliu) [inline docs/quicktour.mdx:136]:
```suggestion
2. Run the following command to launch the example script:
```

File: docs/quicktour.mdx [hunk 5]
```diff
@@ -0,0 +1,300 @@
...
+  ```bash
+  accelerate launch --config_file ds_zero3_cpu.yaml examples/peft_lora_seq2seq_accelerate_ds_zero3_offload.py
+  ```
+
+  c. output logs:
```
Comment (stevhliu) [inline docs/quicktour.mdx:141]:
```suggestion
You'll see some output logs that look like this:
```

**Review by stevhliu — COMMENTED** [review]
Wow you're so fast, thanks for kicking this off! I think the main thing is the actual docs need to be in a `source` file, for example `docs/source/index.mdx`.

File: docs/index.mdx [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+Seamlessly integrated with 🤗 Accelerate for large scale models leveraging DeepSpeed and Big Model Inference. 
+
+Supported methods, with more coming soon:
+
+1. LoRA: [LORA: LOW-RANK ADAPTATION OF LARGE LANGUAGE MODELS](https://arxiv.org/pdf/2106.09685.pdf)
+2. Prefix Tuning: [Prefix-Tuning: Optimizing Continuous Prompts for Generation](https://aclanthology.org/2021.acl-long.353/), [P-Tuning v2: Prompt Tuning Can Be Comparable to Fine-tuning Universally Across Scales and Tasks](https://arxiv.org/pdf/2110.07602.pdf)
+3. P-Tuning: [GPT Understands, Too](https://arxiv.org/pdf/2103.10385.pdf)
+4. Prompt Tuning: [The Power of Scale for Parameter-Efficient Prompt Tuning](https://arxiv.org/pdf/2104.08691.pdf) 
+
+## Getting started
```
[the remark this answers is not in the document]
Comment (MKhalusova) [reply docs/index.mdx:28]: +1 to moving the example, plus it likely needs some context and a description of what's going on in the example.

**Review by MKhalusova — APPROVED** [review]
Great start!  I think that "quick tour" is a bit overwhelming, and maybe there's a way to split it further or focus on simple examples, and leave the details and caveats for other sections.

**Review by stevhliu — APPROVED** [review]
Thanks for all the changes! Last thing is to just put all the docs under a `docs/source` folder 👍

File: docs/quicktour.mdx [hunk 7]
```diff
@@ -0,0 +1,300 @@
...
+Please refer to the [Config Page](package_reference/config) for more details. 
+Below, we will use `LoRAConfig` for demonstration.
+
+```python
+from peft import LoraConfig, TaskType
+
+peft_config = LoraConfig(task_type=TaskType.SEQ_2_SEQ_LM, inference_mode=False, r=8, lora_alpha=32, lora_dropout=0.1)
+```
+
+Here, `task_type` is the type of task you are training your model for.
```
[the remark this answers is not in the document]
Comment (pacman100) [reply docs/quicktour.mdx]: Done.

File: docs/index.mdx (hunk 6 above)
[the remark this answers is not in the document]
Comment (pacman100) [reply docs/index.mdx]: Done.

File: docs/package_reference/peft_model
[the remark this answers is not in the document]
Comment (pacman100) [reply docs/package_reference/peft_model:1]: Thank you!

**Review by stevhliu — APPROVED** [review]
Nice, let's merge it! 🚀

**Pull request merged**